### DISCUSSION TITLE
Nastavení odesílatele e-mailu

### DISCUSSION THREAD

**Discussion by lucienevludova-direct** [canarytrace/documentation#211] [Listener]
Ahoj, 

hodilo by se nám mít možnost upravit proměnnou pro odesílatele e-mailu. Nyní se e-mail odesílá vždy z adresy notifications@canarytrace.com. Jelikož si tyto informace (celý e-mail) předáváme do kanálu v aplikaci MS Teams, tak bychom potřebovali mít možnost nastavit adresu tak, aby se e-mail doručil do našeho workspace.

Děkuji a přeji pěkný den.
Lucka

**Comment by rdpanek** [comment]
Ahoj @lucienevludova-direct vyšla aktualizace `listener-agent` 👉 https://github.com/canarytrace/documentation/issues/213#issuecomment-1953941521 která obsahuje možnost nastavit e-mail odesílatele.
